site/src/jsMain/kotlin/com/varabyte/kobweb/site/components/widgets/video/YoutubeVideo.kt

```diff
@@ -0,0 +1,71 @@
+package com.varabyte.kobweb.site.components.widgets.video
+
+
+import androidx.compose.runtime.Composable
+import com.varabyte.kobweb.compose.css.height
+import com.varabyte.kobweb.compose.css.width
+import com.varabyte.kobweb.compose.foundation.layout.Box
+import com.varabyte.kobweb.compose.ui.Alignment
+import com.varabyte.kobweb.compose.ui.Modifier
+import com.varabyte.kobweb.compose.ui.modifiers.padding
+import com.varabyte.kobweb.silk.style.breakpoint.Breakpoint
+import com.varabyte.kobweb.silk.theme.breakpoint.rememberBreakpoint
+import org.jetbrains.compose.web.css.cssRem
+import org.jetbrains.compose.web.dom.Iframe
+
+/**
+ * Youtube Video Player
+ */
+@Composable
+fun YoutubeVideo(url: String) {
```

Comment:
Nit: The official name is capitalized "YouTube", so this should be "YouTubeVideo"

(and apply to any other instance of the word "Youtube" in this commit)